src/vivarium_public_health/utilities.py

```diff
@@ -97,3 +97,16 @@ def is_non_zero(data: Union[Iterable[ScalarValue], ScalarValue, pd.DataFrame]) -
         attribute_sum = data
 
     return attribute_sum != 0.0
+
+
+def get_lookup_columns(
```

Comment:
a bit pedantic, but maybe it would be slightly more consistent to use `required_columns`, as that's what the classes calling this use.

Reply:
I was trying to keep the names different because the required columns for the lookup tables are different functionally than the `required_columns `for the class since those are defined by Component.